.gitignore

```diff
@@ -1 +1,2 @@
 target
+Cargo.lock
```

Comment:
This is because of http://doc.crates.io/guide.html#cargotoml-vs-cargolock As a lib we should not commit this file.